commands/doit.go

```diff
@@ -215,6 +215,7 @@ func computeCmd() *Command {
 	cmd.AddCommand(ReservedIP())
 	cmd.AddCommand(ReservedIPAction())
 	cmd.AddCommand(ReservedIPv6())
+	cmd.AddCommand(BYOIPPrefix())
```

Comment:
i think we should add it in the `doctl network` command instead of `doctl command`

Reply:
but reserved_ip and reserved_ipv6 are under compute